languages/ruby/exercises/concept/numbers/.docs/introduction.md

```diff
@@ -0,0 +1,24 @@
+There are two different types of numbers in Ruby:
+
+- Integers: numbers with no digits behind the decimal separator (whole numbers). Examples are `-6`, `0`, `1`, `25`, `976` and `500000`.
+- Floating-point numbers: numbers with zero or more digits behind the decimal separator. Examples are `-2.4`, `0.1`, `3.14`, `16.984025` and `1024.0`.
+
+They are implemented through the `Integer` and `Float` class.
```

Comment:
We might mention the parent class `Numeric` here.  The concept is "Numbers" so we should explain where they come from, that some of the behavior they have come from the parent class.

Reply:
> We might mention the parent class Numeric here. The concept is "Numbers" so we should explain where they come from, that some of the behavior they have come from the parent class.

I would argue against mentioning that. Knowing that these classes have a shared base means knowing what inheritance inheritance is and how it works, which we don't know for certain that students will be familiar with at this point. I think this information should either be mentioned in another, separate, more advanced numbers exercise (my preference) or otherwise only be mentioned in the `after.md` document as I don't think it is information the student needs to know to be able to solve the exercise.

Reply:
Sorry, some Stroop Effect happens for me, as I keep thinking these are about the concepts, rather than strictly what is needed to solve an exercise.

Because class was mentioned repeatedly, I thought inheritance was covered.

Reply:
No worries.

Reply:
```suggestion
They are implemented through the `Integer` and `Float` classes.
```